website/docs/components/modal/partials/accessibility/accessibility.md

```diff
@@ -6,8 +6,10 @@ The Modal component is conformant when used as directed.
 
 ## Focus and focus order
 
-- When the Modal is triggered via the keyboard, focus is trapped within the Modal. 
+- When the Modal opens, focus moves to an element contained in the Modal.
```

Comment:
Should this be an interactive element? Would we want to give any details on what type of element?

Reply:
I agree, this reads a bit ambiguous to me. There's more clarity that you added below that is useful, I wonder if this bullet point and the next could be combined somehow:

"When the modal is open, focus moves to the first interactive element within the modal and is trapped within the component."

That's not really perfect either, but throwing out some options.